### DISCUSSION TITLE
Question about `fixable` attribute

### DISCUSSION THREAD

**Discussion by sryze** [eslint/eslint#15489] [Q&A] [answered]
What is the difference between `code` and `whitespace` values for `fixable`? I could not find an explanation in the documentation.

I'm not sure which value to use... Is `whitespace` supposed to be used for changes only in spacing/newlines/indentation and `code` for everything else?

**Comment by nzakas** [comment, marked as answer]
> I'm not sure which value to use... Is whitespace supposed to be used for changes only in spacing/newlines/indentation and code for everything else?

Exactly.
